lib/src/google_fonts_base.dart

```diff
@@ -256,7 +260,7 @@ Future<String> get _localPath async {
 
 Future<File> _localFile(String name) async {
   final path = await _localPath;
-  // TODO(clocksmith): what if this file is an otf?
+  // TODO(clocksmith): What if this file is an otf?
```

Comment:
This TODO can be removed, or replaced with a comment that we only ever expect .ttf from the Google Fonts API (which is the only files we write/read from local storage).

This is different from fonts that might be included in the pubspec.